doc/admin-guide/triggers.rst

```diff
@@ -1,6 +1,5 @@
-********
 Triggers
```

Comment:
Is that file easy to generate using comment extraction from PostgreSQL? If you did it that way, can you share the query used in the doc itself in the contributor page?
My concern is to keep such a page up to date in a documentation page.